connectors-home/modules/ROOT/pages/introduction/intro-config-oauth2-cloudhub.adoc

```diff
@@ -0,0 +1,39 @@
+= Configuring OAuth 2.0 for Connectors on CloudHub
+
+You can deploy your OAuth 2.0 enabled connectors to a cloud service. Learn the required configuration for the OAuth 2.0 setup and how to deploy your connector to CloudHub.
+
+== How to Configure OAuth 2.0 for a Connector on CloudHub
+
+Configuring OAuth 2.0 for a connector on CloudHub is the same as configuring OAuth 2.0 for a connector locally, except with CloudHub you are behind an extra server. You must keep in mind the following when configuring OAuth 2.0 for a connector on CloudHub:
+
+* Connector Listener
++
+** CloudHub manages some xref:runtime-manager::cloudhub-manage-props.adoc#cloudhub-reserved-properties[properties] for you automatically, such as `${http.port}` and `${https.port}`. Use these properties when configuring your *HTTP Listener*.
++
+image::intro-config-oauth2-cloudhub-img1.png[Http Listener configuration window for CloudHub example]
++
+** When deployed on the cloud, the `localhost`, `0.0.0.0`, and `127.0.0.1` hosts are no longer publicly reachable. When configuring your OAuth provider, you must replace all links containing these hosts with the actual CloudHub provided public addresses. For CloudHub, this address is in the format `{ch-app-name}.{region}.cloudhub.io`. For example, an HTTP application named `ch-example-app` deployed in the Europe (London) xref:runtime-manager::cloudhub-networking-guide.adoc#regional-services[region] has public address of `\http://ch-example-app.uk-e1.cloudhub.io`.
+
+* OAuth Provider Setup
++
+** If you set *Callback Path* to `/callback`, your public callback URL is `{ch-app-name}.{region}.cloudhub.io/callback`. This is the callback URL you want to register with your OAuth provider as your redirect URI.
```

Comment:
This is the callback URL you want to register with your OAuth provider as your redirect URI.

Awkward - Maybe something like this: This is the callback URL to use as your redirect URI when you configure OAuth.